src/config.rs

```diff
@@ -89,8 +88,12 @@ impl Template {
             let (ref key, ref value) = param;
             builder = builder.insert_str(key, value);
         }
+        self.render_data(&builder.build())
```

Comment:
Now that we know this tweak is necessary, would it be feasible to put this hunk in the commit that introduces this code?